protocol/SignalingProtocol.proto

```diff
@@ -304,11 +304,23 @@ message SdkTranscriptItem {
     }
     optional Type type = 6;
     optional bool vocabulary_filter_match = 7;
+    optional string confidence = 8;
+    optional bool stable = 9;
+}
+
+message SdkTranscriptEntity {
+    optional string category = 1;
+    optional double confidence = 2;
+    optional string content = 3;
+    optional int64 end_time = 4;
+    optional int64 start_time = 5;
+    optional string type = 6;
```

Comment:
The `type` field in the `SdkTranscriptItem` message indicates the internal enum `Type`. Recommend using an enum here too if the `type` field is not arbitrary.

Reply:
The type field in entity describes the type of PII data which is identified. For instance, type could be ADDRESS or PHONE or EMAIL or Credit/debit number. It has a wide range of values which could change periodically on the transcribe service. Considering this it would be a breaking change if we add this as enum. If any type values are removed or added to the transcribe entity type in the transcribe service would mean to have that change reflect in the client model as well. Also, the way the PII types are being sent to the transcribe api during start transcription is in form of list of strings so we have kept the type in the incoming stream as string only.

Please let me know if you have any suggestion to this.

Reply:
Thanks for the explanation. If the `type` frequently changes in Amazon Transcribe, the `string` type makes sense.